examples/wave_example.ipynb

```diff
@@ -25,154 +25,1821 @@
-      "5.5   0.650992       NaN  \n",
-      "6.0   0.607072       NaN  \n",
-      "6.5   0.658012       NaN  \n",
-      "7.0   0.701503       NaN  \n",
-      "7.5   0.609165       NaN  \n",
-      "8.0        NaN       NaN  \n",
-      "8.5        NaN       NaN  \n",
-      "9.0        NaN       NaN  \n",
-      "9.5        NaN       NaN  \n",
-      "10.0       NaN       NaN  \n",
-      "10.5       NaN       NaN  \n",
-      "11.0       NaN       NaN  \n"
+      "[743 rows x 1 columns]\n"
      ]
     }
    ],
    "source": [
-    "# Calculate capture length\n",
-    "L = wave.performance.capture_length(P, J) \n",
-    "\n",
-    "# Generate bins for Hm0 and Te\n",
-    "Hm0_bins = np.arange(0, Hm0.max() + .5, .5) # input is start, stop, and step size\n",
-    "Te_bins = np.arange(0, Te.max() + 1, 1)\n",
-    "\n",
-    "# Create capture length matrices using mean, standard deviation, count, min and max statistics\n",
-    "LM_mean = wave.performance.capture_length_matrix(Hm0, Te, L, 'mean', Hm0_bins, Te_bins)\n",
-    "LM_std = wave.performance.capture_length_matrix(Hm0, Te, L, 'std', Hm0_bins, Te_bins)\n",
-    "LM_count = wave.performance.capture_length_matrix(Hm0, Te, L, 'count', Hm0_bins, Te_bins)\n",
-    "LM_min = wave.performance.capture_length_matrix(Hm0, Te, L, 'min', Hm0_bins, Te_bins)\n",
-    "LM_max = wave.performance.capture_length_matrix(Hm0, Te, L, 'max', Hm0_bins, Te_bins)\n",
-    "\n",
-    "# Print capture length matrices using mean\n",
-    "print(LM_mean)"
+    "h = 60                                                  # Set water depth to 60 m\n",
+    "J = wave.resource.energy_flux(ndbc_data,h)              # Compute the energy flux from spectra and water depth\n",
+    "                                                      \n",
+    "Hm0 = wave.resource.significant_wave_height(ndbc_data) # Compute significant wave height from spectra and \n",
+    "                                                      \n",
+    "print(J)\n",
+    "print(Hm0)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Additional capture length matrices can be computed, for example, the frequency matrix is computed below."
+    "### Conversion from DataFrames to Series\n",
+    "MHKiT functions typically take in Pandas Series so that the function knows exactly which column to use. Pandas Series are one-dimensional ndarrays with axis labels which are most commonly encountered when a single column is requested from a DataFrame (e.g. `ndbc_data['2018-01-01 01:40:00']` returns a Pandas Series from the Pandas DataFrame of all frequencies at the specified time). \n",
+    "\n",
+    "To the above results (energy Period, energy flux, and significant wave height) were returned as DataFrames. For DataFrames which only have one column the conversion from DataFrame to Series can accomplished using the `squeeze` method alternatively we can call on the column header. Each method is demonstrated below:"
```

Comment:
@rpauly18 So my major contribution was here where I separated out each of the previous squeeze calls to explain better why we were doing it and how a user could generate a Series.

Reply:
@ssolson is there a reason why the Capture Length Matrix section was removed? That's an important part of the wave example, and an important result of the IEC -100 specification.

Reply:
Can you show me where I removed this in 5667273fa952633cf922f816cf42553113142e28?

Reply:
I don't see it in that commit, it looks like the "Capture Length Matrices" section was already removed by your commit. I cannot see when/how it was removed.

Reply:
If you believe it should be there I think it is pretty obvious that it should be. I did not remove this as far as I can tell. Can you replace the section in the most logical place based on the current commit?

Reply:
My apologies. The section was not removed. I'm not sure why the commit history in this PR made it look like it had been. Long story short, when reviewing a PR fora `*.ipynb` file on GitHub I recommend looking at the "display the rendered blob" and not the "display the source blob". 

![image](https://user-images.githubusercontent.com/4603379/83546187-87a91200-a4bd-11ea-9494-2f18cd2ef8b9.png)